test/spec/all.js

```diff
@@ -26,3 +28,4 @@ import './directives/filereader.spec.js';
 import './directives/scaleselector.spec.js';
 import './directives/layertree.spec.js';
 import './directives/grid.spec.js';
+import '../../contribs/gmf/test/spec/all.js';
```

Comment:
Why do you add a dependency on gmf files?

Reply:
Finally to don't have tow times Angular in the test scripts...